src/DynamoCoreWpf/UI/GuidedTour/GuideFlowEvents.cs

```diff
@@ -50,6 +50,7 @@ internal static void OnGuidedTourFinish(string name)
             {
                 isAnyGuideActive = false;
                 GuidedTourFinish(new GuidedTourStateEventArgs(name));
+                Logging.Analytics.TrackEvent(Logging.Actions.Completed, Logging.Categories.GuidedTourOperations, name);
```

Comment:
The `name` here is actual tour name, so if we want to be consistent, we may want to also use the real tour name above for next step